### PR TITLE
Convert to 1.9 hash syntax

### PR DESCRIPTION
**Pull request by camertron**
Mind talking a look, @KL-7?

### CODE REVIEW

File: lib/twitter_cldr.rb [hunk 1]
```diff
@@ -41,10 +41,10 @@ module TwitterCldr
   # TODO: convert this and all other hashes to 1.9 syntax
   # maps twitter locales to cldr locales
   TWITTER_LOCALE_MAP = {
-      :msa     => :ms,
+      msa:     :ms,
       :'zh-cn' => :zh,
       :'zh-tw' => :'zh-Hant',
-      :no      => :nb
+      no:      :nb
```
Comment (KL-7) [inline lib/twitter_cldr.rb]: If the whole hash can't be converted to the new syntax, I usually leave it with the old one entirely. I think mixing the two doesn't look that good.

**Comment by KL-7** [comment]
Nice! I quickly scanned through the diff and didn't spot anything suspicious, except for the first file.

**Comment by KL-7** [comment]
This will resolve #151 :+1:

File: lib/twitter_cldr.rb (hunk 1 above)
Comment (camertron) [inline lib/twitter_cldr.rb]: Ok good call, thanks!

**Comment by camertron** [comment]
Yeah, I thought it was about time we finally did this :)

File: lib/twitter_cldr/resources/tailoring_importer.rb [hunk 2]
```diff
@@ -25,17 +25,17 @@ class TailoringImporter < IcuBasedImporter
 
       LOCALES_MAP = {
         :'zh-Hant' => :'zh_Hant',
```
Comment (KL-7) [inline lib/twitter_cldr/resources/tailoring_importer.rb]: Uh, here's another mixed hash. I think that's the last one.

**Comment by KL-7** [comment]
:shipit: once that other hash is reverted :hamster:

File: lib/twitter_cldr/resources/tailoring_importer.rb (hunk 2 above)
Comment (camertron) [inline lib/twitter_cldr/resources/tailoring_importer.rb]: Nice catch!

**Pull request merged**